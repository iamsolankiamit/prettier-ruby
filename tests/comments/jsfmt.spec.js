run_spec(__dirname, ["ruby"]);
