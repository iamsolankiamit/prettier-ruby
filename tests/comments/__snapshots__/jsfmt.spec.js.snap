// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comments.rb 1`] = `
alias $foo $bar #comment

alias store []= #comment 2

alias :foo :bar #comment test test test test

alias foo bar
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
alias $foo $bar # comment

alias store []= # comment 2

alias :foo :bar # comment test test test test

alias foo bar

`;
